stable/pomerium/README.md

```diff
@@ -75,6 +75,9 @@ Parameter                         | Description
 `service.annotations`             | Service annotations                                                                                                                                                                                        | `{}`
 `service.externalPort`            | Pomerium's port                                                                                                                                                                                            | `443`
 `service.type`                    | Service type (ClusterIP, NodePort or LoadBalancer)                                                                                                                                                         | `ClusterIP`
+`serviceMonitor.enabled`          | Create Prometheus Operator ServiceMonitor                  | `false`
```

Comment:
it's more about cosmetic than semantics, but on all the other lines, the example cell is on its own line :)